pkg/storage/kube/build.go

```diff
@@ -34,7 +39,49 @@ func (s *store) GetBuild(id string) (*brigade.Build, error) {
 	return b, err
 }
 
-// Get creates a new project and writes it to storage.
+// DeleteBuild deletes a build.
+func (s *store) DeleteBuild(bid string, options storage.DeleteBuildOptions) error {
+	opts := meta.ListOptions{
+		LabelSelector: fmt.Sprintf(jobFilter, bid),
+	}
+	delOpts := meta.NewDeleteOptions(0)
+	pods, err := s.client.CoreV1().Pods(s.namespace).List(opts)
+	if err != nil {
+		return err
+	}
+	if options.SkipRunningBuilds {
+		for _, p := range pods.Items {
+			if p.Labels["component"] == "build" {
+				if p.Status.Phase == v1.PodRunning || p.Status.Phase == v1.PodPending {
+					log.Printf("skipping Build %s because it is on Status %s", p.Labels["build"], p.Status.Phase)
```

Comment:
minor nit on wording/case here... perhaps `skipping build <build id> because its status is <status>`?

Reply:
just added that, thanks!